lake_envs.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+"""Defines some frozen lake maps."""
+from gym.envs.toy_text import frozen_lake, discrete
+from gym.envs.registration import register
```

Comment:
According to Mo, avoid statements like `from foo import bar`. It's better to use the fully qualified function name like `foo.bar` so it's easier to read.